cranelift/filetests/filetests/isa/x64/simd-comparison-legalize.clif

```diff
@@ -55,14 +55,16 @@ block0(v0: i16x8, v1: i16x8):
 ;   Entry block: 0
 ; Block 0:
 ;   (original IR block: block0)
-;   (instruction range: 0 .. 7)
+;   (instruction range: 0 .. 9)
 ;   Inst 0:   pushq   %rbp
 ;   Inst 1:   movq    %rsp, %rbp
-;   Inst 2:   pminsw  %xmm1, %xmm0
-;   Inst 3:   pcmpeqw %xmm1, %xmm0
```

Comment:
If we end up merging this as-is, I'll add an issue to remind us to make sure these extra moves go away in the new regalloc.